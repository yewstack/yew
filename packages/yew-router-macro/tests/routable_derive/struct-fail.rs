#[derive(yew_router::Routable)]
struct Test {}

fn main() {}
