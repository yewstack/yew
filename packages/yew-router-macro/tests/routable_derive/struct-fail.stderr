error: expected enum, found struct
 --> $DIR/struct-fail.rs:2:1
  |
2 | struct Test {}
  | ^^^^^^
